### PR TITLE
Km/replace non number to blank

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium/pull/11503#pullrequestreview-162990982

### CODE REVIEW

File: lib/tools/adb-emu-commands.js [hunk 1]
```diff
@@ -130,7 +130,7 @@ emuMethods.sendSMS = async function (phoneNumber, message = '') {
   if (message === "") {
     throw new Error('Sending an SMS requires a message');
   }
-  phoneNumber = `${phoneNumber}`.replace(/\s*/, "");
+  phoneNumber = `${phoneNumber}`.replace(/\D/g, "");
```
Comment (mykola-mokhnach) [inline lib/tools/adb-emu-commands.js]: do we also want to replace the leading `+`?

File: lib/tools/adb-emu-commands.js [hunk 2]
```diff
@@ -149,7 +149,7 @@ emuMethods.gsmCall = async function (phoneNumber, action = '') {
   if (_.values(emuMethods.GSM_CALL_ACTIONS).indexOf(action) === -1) {
     throw new Error(`Invalid gsm action param ${action}. Supported values: ${_.values(emuMethods.GSM_CALL_ACTIONS)}`);
   }
-  phoneNumber = `${phoneNumber}`.replace(/\s*/, "");
+  phoneNumber = `${phoneNumber}`.replace(/\D/g, "");
```
Comment (mykola-mokhnach) [inline lib/tools/adb-emu-commands.js]: we usually use single quotes

**Comment by KazuCocoa** [comment]
the command can work with non numbers like below. I think we can close this..

`adb emu sms send '+81 0820 22-0000' hi` 

image

**Comment by jlipps** [comment]
oh, n/m just saw your other comment about things still working.

**Pull request closed without merging**